### ISSUE TITLE
mesheryctl inaccurately identifies a missing prerequisite: docker-compose

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery#253]
**Description**
My system (a Mac) has Docker Desktop installed and running. When I execute `mesheryctl status`, I get an error saying that `docker-compose` is not installed. This prerequisite check is incorrect.

```
./mesheryctl status


Meshery containers status . . .

2019/09/12 09:29:23 [ERROR] Please, install docker-compose. The error message:
exit status 1
```

**Comment by leecalcote** [comment]
Best way to check for presence of `docker-compose`? A `which docker-compose`? Presence of a file in `/usr/bin/` or in `/usr/local/bin`? Suggested solution is to run `docker-compose version`, which will serve the purpose of both confirming that `docker-compose` works and  guaranteeing compatibility with the version that Meshery needs.

**Comment by leecalcote** [comment]
Thanks, @nunix!

**Comment by leecalcote** [comment]
Fixed by #239 #240
